docker/debian11/Dockerfile-jormungandr

```diff
@@ -0,0 +1,55 @@
+FROM debian:bullseye-slim
+
+WORKDIR /usr/src/app
+
+COPY ./source/navitiacommon ./navitiacommon
+COPY ./source/jormungandr ./jormungandr
+COPY ./docker/ca-certificates/*.crt /usr/local/share/ca-certificates/
+
+RUN apt-get update --fix-missing \
+    && apt-get install -y curl libpq5 apache2 python3.9-dev python3-pip git libgeos-c1v5 ca-certificates \
+    && update-ca-certificates \
+    && (cd navitiacommon && python3 setup.py install) \
+    && (cd jormungandr && python3 setup.py install && pip3 install --no-cache-dir -U -r requirements.txt)\
+    && pip3 install --no-cache-dir uwsgi==2.0.21 \
+    && rm -rf navitiacommon jormungandr \
+    && apt-get purge -y \
+        python3-pip \
+        git \
+    && apt-get autoremove -y
+
+
+COPY ./docker/run_jormungandr.sh ./run.sh
+COPY ./docker/jormungandr.wsgi ./jormungandr.wsgi
+
+# Add apache config
+RUN rm /etc/apache2/sites-available/000-default.conf /etc/apache2/conf-available/other-vhosts-access-log.conf
+COPY ./docker/apache/jormungandr.default.conf /etc/apache2/sites-available/000-default.conf
+COPY ./docker/apache/apache2.conf /etc/apache2/apache2.conf
+COPY ./docker/apache/envvars /etc/apache2/envvars
+COPY ./docker/apache/logformat.conf /etc/apache2/conf-available/logformat.conf
+COPY ./docker/apache/security.conf /etc/apache2/conf-available/security.conf
+
+RUN a2ensite 000-default.conf \
+    && a2enmod rewrite \
+    && a2enmod proxy \
+    && a2enmod proxy_balancer \
+    && a2enmod proxy_http \
+    && a2enconf logformat.conf \
+    && a2enmod deflate
+
+ # Python 'requests' package handle its own CA certificate list
+ # Let's force it to use the OS's list
+ ENV REQUESTS_CA_BUNDLE /etc/ssl/certs
+
+# Redirect apache log output to docker log collector
+ RUN ln -sf /proc/self/fd/1 /var/log/apache2/access.log \
+     && ln -sf /proc/self/fd/2 /var/log/apache2/error.log \
+     && echo "net.ipv4.tcp_tw_reuse=1" >> /etc/sysctl.d/10-ipv4.conf
```

Comment:
Could you add a comment explaining why we set `net.ipv4.tcp_tw_reuse=1` ? 
Otherwise we will quickly forget why :)

Reply:
tcp_tw_reuse allows the reuse of a client port immediately after the connection has been closed, even though the connection is still in TIME_WAIT state. It's important to test the configuration before to push it in production.